app/src/main/java/me/hufman/androidautoidrive/music/QueueMetadata.kt

```diff
@@ -0,0 +1,9 @@
+package me.hufman.androidautoidrive.music
+
+import android.graphics.Bitmap
+
+data class QueueMetadata(val title: String? = null,
+                         val subtitle: String? = null,
+                         val songs: List<MusicMetadata>? = null,
+						 val coverArt: Bitmap? = null
```

Comment:
Use spaces here to match the other lines

Reply:
This should be fixed